### ISSUE TITLE
Shall we split data in separate package?

### ISSUE THREAD

**Issue by YoungFaithful** [YoungFaithful/CapacityExpansion.jl#29]
- The example data is currently provided automatically with the package
- The data has 150 MB, the other files 4.2 MB
- We could host the example data in a separate repository that is a julia package as well [JouliaExamples](https://github.com/JuliaEnergy/JouliaExamples)

**Comment by YoungFaithful** [comment]
@holgerteichgraeber what do you think is the best practice?

**Comment by holgerteichgraeber** [comment]
Good point. I think it would be good practice to separate the two. However, I am not sure if at this point it would make any difference. The data is already in the git history, i.e. even if we take it out, I think that the package will still be 150MB large if you clone it from git.

**Comment by holgerteichgraeber** [comment]
Move data to CapacityExpansionData. The git is only cloned for dev, when using it as a package, git history is not downloaded.

**Comment by YoungFaithful** [comment]
https://github.com/YoungFaithful/CapacityExpansionData.jl/
We can continue removing the `/data/` folder as soon as CapacityExpansionData is registered.
